docs/docker.md

```diff
@@ -8,13 +8,134 @@ Start by downloading and installing Docker CE for your platform:
 * [Windows](https://docs.docker.com/docker-for-windows/install/)
 * [Linux](https://docs.docker.com/install/)
 
+Optionally, [docker-compose](https://docs.docker.com/compose/install/) should be installed and available to follow the docker quick start guide.
+
 Once you have Docker installed, simply prepare the config file (e.g. `config.json`) and run the image for `freqtrade` as explained below.
 
-## Download the official FreqTrade docker image
+## Freqtrade with docker-compose
+
+Freqtrade provides an official Docker image on [Dockerhub](https://hub.docker.com/r/freqtradeorg/freqtrade/), as well as a [docker-compose file](https://github.com/freqtrade/freqtrade/blob/develop/docker-compose.yml) ready for usage.
+
+!!! Note
+    The following section assumes that docker and docker-compose is installed and available to the logged in user.
+
+!!! Note
+    All below comands use relative directories and will have to be executed from the directory containing the `docker-compose.yml` file.
+
+### Docker quick start
+
+Create a new directory and place the [docker-compose file](https://github.com/freqtrade/freqtrade/blob/develop/docker-compose.yml) in this directory.
+
+``` bash
+mkdir freqtrade
```

Comment:
this may fail if user is in the freqtrade main dir,
maybe use `freqtrade-compose` or `user_data/docker-compose` dir here? (the latter is also not good if NOT in freqtrade main dir)
or `~/freqtrade-compose` ?

Reply:
well my assumption for this is that a user did NOT clone the repository - but starts from scratch.

Cloning the repository is no really necessary when using the docker image (it's also not necessary when installing from pypi - but that section of the docs will still need "some" cleanup).

Reply:
another reason for using another directory name:
there are lots of commands here with 'freqtrade' in them, like
```
docker-compose run --rm freqtrade create-userdir --userdir user_data
```
So if a user want to change the name of the dir, should she change 'freqtrade' in the above command as well? This can be a question for an non-experienced user...

Anyway, so far so good.